### PR TITLE
Revise run_pipeline comment (as expected)

### PR DESCRIPTION
**Pull request by jingzhang36**
As part of the migration effort, when a run is created with both pipeline id and version id, version id takes precedence. Specifically, in the backend, when a run is to be created, we try to acquire workflow manifest from specified pipeline version first; if fail, we then try to acquire it from pipeline id's default version.

https://github.com/kubeflow/pipelines/blob/6e56960e6a7bd0ba5e95bf13d10477b8923f4789/backend/src/apiserver/resource/resource_manager_util.go#L236

### CODE REVIEW

File: sdk/python/kfp/_client.py [hunk 1]
```diff
@@ -386,9 +386,9 @@ def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, para
       pipeline_package_path: local path of the pipeline package(the filename should end with one of the following .tar.gz, .tgz, .zip, .yaml, .yml).
       params: a dictionary with key (string) as param name and value (string) as as param value.
       pipeline_id: the string ID of a pipeline.
-      version_id: the string ID of a pipeline version. 
-        If both pipeline_id and version_id are specified, pipeline_id will take precendence
-        This will change in a future version, so it is recommended to use version_id by itself
+      version_id: the string ID of a pipeline version.
+        If both pipeline_id and version_id are specified, version_id will take precendence
```
Comment (numerology) [inline sdk/python/kfp/_client.py:390]: nit: period after sentence.

Comment (jingzhang36) [reply sdk/python/kfp/_client.py]: added.

**Pull request merged**